CHANGELOG

```diff
@@ -1,3 +1,72 @@
+1.5.0.rc1
+  # general
+  - You can now use LS_HOME/patterns directory to add generic patterns for those that may not be
+    associated with a particular plugin. Patterns in this dir will be loaded by default (#2225)
+  - We now check if the config file is correctly encoded. Otherwise we show a verbose error message
+    to convert the failing config file(s) to UTF-8 (#LOGSTASH-1103)
+  - Fixed bug in pipeline to gracefully teardown output workers when num workers > 1 (#2180)
+  - Fixed nologin path in release debian packages (#2283)
+  - Resolved issue where Logstash was crashing for users still using exclude_tags in their output
+    configuration (#2323)
+  - Updated and releases a new version of the logstash-event gem with latest changes in 1.5.0 RC1
+  - Windows: Significantly improved the initial user experience with Windows platform (#2504, #1426). 
+    Fixed many issues related to File input. Added support for using the plugin 
+    framework (installing, upgrading, removing)
+  - With the release of Kibana 4, we have removed the `bin/logstash web` command and any reference to
+    Kibana from Logstash
+
+  # input
+  - Added new CouchDB input plugin to fetch data from CouchDB. Using the _changes API, data can be kept
+    in sync with any output like Elasticsearch by using this input
+  - File: Fixed a number of issues on Windows platform. These include:
+    - Resolving file locking issues which was causing log files to not rotate (#1557, #1389)
+    - Added support for using SinceDB to record multiple files' last read information (#1902)
+    - Fixed encoding issues which applies to many inputs (#2507)
+    - Resolved Logstash skipping lines when moving between files which are being followed (#1902)
+  - Kafka: Added support for whitelisting and blacklisting topics in the input. 
+  - EventLog: For Windows, this input gracefully shutsdown if there is a timeout while receiving events
+    This also prevents Logstash from being stuck (#1672)
+  - Heartbeat: We created a new input plugin for generating heartbeat messages at periodic intervals. 
+    Use this to monitor Logstash -- you can measure the latency of the pipeline using these heartbeat events,
+    and also check for availability
+  - S3: 
+    - Added IAM roles support so you can securely read and write events from S3 without providing your
+      AWS credentials (#1575). 
+    - Added support for using temporary credentials obtained from AWS STS (#1946)
+  - Lumberjack: Resolved issue where unrelated events were getting merged into a single event while using
+    this input with with the multiline codec (#2016)
+
+  # filter
+  - Mutate: 
+    - Resolved issue where convert option was creating an extra field in the event (#2268)
+    - Fixed issue where mutate with non-existent field was throwing an error (#2379)
+  - Multiline: Handled cases where we dropped lines from certain multiline xml messages (#3) 
```

Comment:
maybe "where we unintentionally deduplicated lines, such as repeated lines in xml messages"